README.md

```diff
@@ -58,4 +58,36 @@ SBT depedency:
 
 ````scala
 libraryDependencies += "com.softwaremill.common" %% "id-generator" % "1.0.0"
+````
+
+## Future Try extensions
+
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.softwaremill.common/futuretry_2.11/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.softwaremill.common/futuretry_2.11)
+
+Provides two utility methods for extending `Future`:
+
+ - `tried: Future[Try[T]]` - reifying the Future's result.
+ - `transTry(f: Try[T] => Try[S]): Future[S]` - corresponds to 2.12's new `transform` variant, allowing to supply a single function (if, for example, you already have one handy), 
```

Comment:
I also prefer meaningful names. And if you included such a lib, it means you want to use functions defined in it so you will know that there is `transformTry` function available. You can point this out during code review if someone missed out the function.